pkg/reconciler/wasmmodule/wasmmodule_test.go

```
@@ -0,0 +1,91 @@
+	if err := json.Unmarshal([]byte(got), &gotMap); err != nil {
+		t.Fatalf("unmarshal got: %v", err)
+	}
+	if err := json.Unmarshal(goldenBytes, &wantMap); err != nil {
+		t.Fatalf("unmarshal golden: %v", err)
+	}
+
+	gotJSON, err := json.MarshalIndent(gotMap, "", "  ")
+	if err != nil {
+		t.Fatalf("marshal got normalized JSON: %v", err)
+	}
+	wantJSON, err := json.MarshalIndent(wantMap, "", "  ")
```

golangci-lint (2.5.0)

[error] missing whitespace above this line (invalid statement above assign)

(wsl_v5)